wurst/lib/ItemRestriction.wurst

```diff
@@ -0,0 +1,393 @@
+/*
+*  ItemRestriction v1.0.0.0
+*     by Bannar
+*
+*  For restricting or limiting items from being equipped.
+*/
+package ItemRestriction
+import HashMap
+import LinkedList
+import UnitIndexer
+import RegisterEvents
+
+HashMap<int, LinkedList<ItemRestriction>> restrictionsMap
+
+@configurable function getUnitTypeErrorMessage(int _unitId) returns string
+    return "This item can not be hold by this unit type."
+
+@configurable function getLevelErrorMessage(int level) returns string
+    return "This item requires level " + level.toString() + " to be picked up."
+
+@configurable function getStatisticErrorMessage(int value, string statistic) returns string
+    return "This item requires " + value.toString() + " " + statistic + "."
+
+@configurable function getLimitErrorMessage(int limit) returns string
+    return "This unit can not hold more than " + limit.toString() + " items of this item type."
```

Comment:
Minor nitpick, but the `s` in `items` will need to be dropped if `limit == 1`.

Reply:
Thanks!